projects/ion/src/lib/table/table.component.spec.ts

```diff
@@ -164,6 +164,53 @@ describe('IonTableComponent', () => {
   });
 });
 
+describe('Table > Changes', () => {
+  const propsToChange: IonTableProps<Disco> = {
+    config: {
+      data: [{ name: 'Blink 182', deleted: false, id: 2 }],
+      columns,
+    },
+  };
+
+  it('should change data in table', async () => {
+    const { rerender } = await render(IonTableComponent, {
+      componentProperties: propsToChange,
+      imports: [
+        FormsModule,
+        IonButtonModule,
+        IonIconModule,
+        IonCheckboxModule,
+        IonPaginationModule,
+        IonTagModule,
+        IonPopConfirmModule,
+      ],
+    });
+    const newData = [{ name: 'Meteora', deleted: false, id: 2 }];
+    propsToChange.config.data = [...newData];
+
+    rerender(propsToChange);
```

Comment:
I was never able to use this function, it never worked properly. I'm gonna use this tests for reference someday ^^